docs/cookbook-recipes/isle-v140-git-cleanup.md

```diff
@@ -0,0 +1,161 @@
+---
+
+## Git cleanup process
+
+
+### Git cleanup process - Overview
+
+The steps below in the section `Git cleanup process` will have you perform the following:
+
+* Open up a terminal and perform steps as directed
+- Make a directory called `ISLE_140_GIT_CLEANUP` on your laptop or workstation to perform all of these steps and collect all materials
+* Download the git cleanup script (`isle-v140-git-cleanup-script.sh`)and move it to the `ISLE_140_GIT_CLEANUP` directory
+- `git clone --mirror` your forked or cloned ISLE project directory to the `ISLE_140_GIT_CLEANUP` directory
+* Run the git cleanup script. It will prompt you to only enter the name of your forked or cloned ISLE project directory.
+- `git push` back the changes to your remote forked or cloned ISLE project directory.
+* Clone down your newly changed and smaller forked or cloned ISLE project directory again to ensure that the repositiory is smaller to a location of your choice.
+- Remove all previous larger size versions of your forked or cloned ISLE project directory. You can now delete the `ISLE_140_GIT_CLEANUP` directory and its contents.
+* Work from the newly resized and smaller forked or cloned ISLE project directory.
+  
+### Git cleanup process - Steps
+
+* Open up a terminal
+
+* Change directory to a location of your choice
+  * `cd your_path_here`
+
+* Make a new temporary direcotory called `ISLE_140_GIT_CLEANUP`
+    * `mkdir ISLE_140_GIT_CLEANUP`
+
+* Download the script found and save it to the `ISLE_140_GIT_CLEANUP` directory and fix the permissions.
+  * Right click on this link below and click "Save link as"
+    * [isle-v140-git-cleanup-script.sh](../assets/isle-v140-git-cleanup-script.sh)
+  * `chmod 755 isle-v140-git-cleanup-script.sh`
+
+* Run this git command to clone your forked or cloned ISLE project directory. This is not the usual `git clone` command, note the `--mirror` flag.
+  * `git clone --mirror git project url`
+  * Example:
+    * `git clone --mirror git@github.com:username/ISLE.git` (_as a fork using the same project name or title_)
+    * `git clone --mirror git@github.com:username/acmeorg_isle.git` (_as a fork or cloned repo using a different project name or title_)
+  * This will create a new directory called `ISLE.git` or `acmeorg_isle.git` which is not typical clone or fork output.
+
+* Run the cleanup script
+  * `./isle-v140-git-cleanup-script.sh`
+
+* At the start of the script, there will be an interactive input prompt asking you to:
+  
+```bash
+"Type in the name of your git repo only e.g. ISLE (do not include .git) and then press the [ENTER] or [RETURN] key to continue"`
+Enter the name here:  
+```
+* Enter the name of your forked or cloned ISLE git repository without the .git extension e.g `ISLE` or `acmeorg-isle` etc.
+
+* The script will continue and the process takes about 30 - 45 seconds depending on the speed of your system with minimal output.
+
+* Your final step is to push this local smaller repo back to the remote repository
```

Comment:
I'd love to figure out how to get a link here, so the message at the end of the script can bring you directly to this point.

Reply:
@bseeger There is a message at the end of the script that says the following: "This script has now completed but there are more steps. Please continue to read the directions at https://islandora-collaboration-group.github.io/ISLE/cookbook-recipes/isle-v140-git-cleanup/"

Reply:
@g7morris  - I was thinking more directly to this step (like an anchor) so they know where to continue from. But presumably they started on this page and are at this point anyways, so they know where they left off.

Reply:
Let's leave this as is for now, as I think folks will have the page open and working through the steps as they go.